src/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
     <meta charset="utf-8">
-    <title>AGAIN Updated src/index.html</title>
+    <title>I beat Charlotte to it</title>
```

Comment:
Why did you call it this? Name the tab what it is trying to achieve? Duh

Reply:
Needs work